nfcore_custom.config

```
@@ -23,6 +23,7 @@ profiles {
   crick        { includeConfig "${params.custom_config_base}/conf/crick.config" }
   czbiohub_aws { includeConfig "${params.custom_config_base}/conf/czbiohub_aws.config" }
   ebc          { includeConfig "${params.custom_config_base}/conf/ebc.config" }
+  eddie	  { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```

gah, still one more..

```suggestion
  eddie	 { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```